Dockerfile.dev

```diff
@@ -0,0 +1,46 @@
+ARG RUBY_VERSION=3.3.1
+ARG NODE_VERSION=20.10.0
+
+# Use Node image so we can pull the binaries from here.
+FROM node:$NODE_VERSION as node
+
+# Ruby build image.
+FROM ruby:${RUBY_VERSION}-slim
+
+RUN apt-get update -qq && \
+    apt-get install -y build-essential libssl-dev libpq-dev vim git libsasl2-dev && \
+    rm -rf /var/lib/apt/lists/*
```

Comment:
Do we have Specific versions for the apt-get packages or not needed?

Reply:
It's not needed to specify the versions for these packages. Usually these packages are provided by the SO